contrib/repairStatus.py

```diff
@@ -0,0 +1,199 @@
+  percent_complete = total_successful_range_repairs * 100 / total_to_repair
+  print ("Checking repair messages for", node, "for the past", days, "days")
+  print ("Count is: " + str(total_successful_range_repairs))
+  print ("Total to repair is: " + str(total_to_repair))
+  print ("Percent Complete: " + str(percent_complete) + "%")
+
+
+def main():
+  parser = argparse.ArgumentParser(
+    description = '''This script shows the current status of a repair running on a Cassandra node.  
+                    Elasticsearch is required so that we can query efficiently.
+
+                    Note that NODE is required, and it should be any part of the hostname
+                    of the node that will uniquely identify the node.''')
+  parser.add_argument('-n', '--node',     type=str,
+                                          required=True, 
+                                          help='''Which node to check. This can be any portion 
+                                                  of the hostname that uniquely identifies the node. 
+                                                  (required)''')
+  parser.add_argument('--host',           type=str, 
+                                          help='Hostname for Elasticsearch (required)', 
+                                          default='localhost')
+  parser.add_argument('-p', '--port',     type=int, 
+                                          help='''Port on host where Elasticsearch is running 
+                                                  (default: 9200)''', 
+                                          default=9200)
+  parser.add_argument('-d', '--days',     type=int, 
+                                          help='''How many days of logs to search. If the script fails to return 
+                                                  data, try increasing the default value here 
+                                                  (default: 3)''', 
+                                          default=3)
+  parser.add_argument('-v', '--verbose',  action='store_true',
+                                          help='Display verbose information')
+  
+  args = parser.parse_args()
+
+  if args.verbose:
+      log.basicConfig(format="%(levelname)s: %(message)s", level=log.DEBUG)
+  else:
+      log.basicConfig(format="%(levelname)s: %(message)s")
+  
+  # Set the current time and the point in time to start searching from
+  current_time=datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.000Z')
+  start_time=(datetime.utcnow() - timedelta(days=args.days)).strftime('%Y-%m-%dT%H:%M:%S.000Z')
+
+  # Calculate how many 
+  total_successful_range_repairs = calculate_range_repairs(args.host, args.port, args.node, start_time, current_time)
+  total_to_repair = calculate_total_ranges(args.host, args.port, args.node, start_time, current_time)
+
+  # Print totals
+  print_totals(args.node, args.days, total_successful_range_repairs, total_to_repair)
+
+
+if __name__ == '__main__': main()
```

Comment:
also a minor coding-style nitpick; most projects have a "no simple suites" rule (a bunch of statements following a ":" is called a "suite") so this would normally be written:
```python
if __name__ == '__main__':
    main()
```